[CIE-5504] c8yedge config docs (#4146)
SoftwareAG/c8y-docs: content/edge-kubernetes/k8s-edge-messaging-service.md

sector:
  - edge_server
---
The {{< product-c8y-iot >}} Messaging Service is an optional component of the {{< product-c8y-iot >}} platform, required for enabling the [microservice-based data broker](/data-broker/ms-data-broker) and [Notifications 2.0](https://{{< domain-c8y >}}/api/core/#tag/About-notifications-2.0). If you installed Edge using the c8yedge tool, you can enable Messaging Service using the `c8yedge config --set messagingService.enabled=true` command. Alternatively, you can enable it by updating the `spec.messagingService` field in the Edge custom resource (CR). For more details on the `spec.messagingService` field, refer to [Edge custom resource - Messaging Service](/edge-kubernetes/edge-custom-resource-definition/#messagingService).

> For additional guidance, see the [Installing Edge](/edge-kubernetes/installing-edge-on-k8/) and [Modifying Edge](/edge-kubernetes/manage-edge/#modify-edge).

Reviewer: I'd suggest replacing with the (much improved) wording you added in register-edge-on-cloud
```suggestion
For general guidance on configuring Edge, see [Modifying Edge](/edge-kubernetes/manage-edge/#modify-edge).
```

---
The {{< product-c8y-iot >}} Messaging Service is an optional component of the {{< product-c8y-iot >}} platform, required for enabling the [microservice-based data broker](/data-broker/ms-data-broker) and [Notifications 2.0](https://{{< domain-c8y >}}/api/core/#tag/About-notifications-2.0).

> To enable the Messaging Service in Edge, update the `spec.messagingService` field in the Edge custom resource (CR). After making the changes, apply the updated CR. For more details on the `spec.messagingService` field, refer to [Edge custom resource - Messaging Service](/edge-kubernetes/edge-custom-resource-definition/#messagingService). You can also enable Messaging Service using `c8yedge config --set messagingService.enabled=true` command.

Reviewer 2: ```suggestion
To enable the Messaging Service in Edge, update the `spec.messagingService` field in the Edge custom resource (CR). After making the changes, apply the updated CR. For more details on the `spec.messagingService` field, refer to [Edge custom resource - Messaging Service](/edge-kubernetes/edge-custom-resource-definition/#messagingService). For installations performed via the c8yedge installer tool, enable the Messaging Service using: `c8yedge config --set messagingService.enabled=true` command.
```

Author: Addressed by https://github.com/Cumulocity-IoT/c8y-docs/tree/36d6143f3ee25faa8242c19754ded55411cb7a6b

> To enable the Messaging Service in Edge, update the `spec.messagingService` field in the Edge custom resource (CR). After making the changes, apply the updated CR. For more details on the `spec.messagingService` field, refer to [Edge custom resource - Messaging Service](/edge-kubernetes/edge-custom-resource-definition/#messagingService). You can also enable Messaging Service using `c8yedge config --set messagingService.enabled=true` command.

Reviewer 3: ```suggestion
To enable the Messaging Service in Edge, update the `spec.messagingService` field in the Edge custom resource (CR). After making the changes, apply the updated CR. For more details on the `spec.messagingService` field, refer to [Edge custom resource - Messaging Service](/edge-kubernetes/edge-custom-resource-definition/#messagingService). You can also enable the Messaging Service using the `c8yedge config --set messagingService.enabled=true` command.
```
